terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -484,7 +484,8 @@ data "aws_iam_policy_document" "oidc_assume_role_member" {
       format("arn:aws:iam::%s:role/modernisation-account-limited-read-member-access", local.environment_management.modernisation_platform_account_id),
       # the two below are required as sprinkler and cooker have development accounts but are in the sandbox vpc
       local.application_name == "sprinkler" ? format("arn:aws:iam::%s:role/member-delegation-garden-sandbox", local.environment_management.account_ids["core-vpc-sandbox"]) : format("arn:aws:iam::%s:role/modernisation-account-limited-read-member-access", local.environment_management.modernisation_platform_account_id),
-      local.application_name == "cooker" ? format("arn:aws:iam::%s:role/member-delegation-house-sandbox", local.environment_management.account_ids["core-vpc-sandbox"]) : format("arn:aws:iam::%s:role/modernisation-account-limited-read-member-access", local.environment_management.modernisation_platform_account_id)
+      local.application_name == "cooker" ? format("arn:aws:iam::%s:role/member-delegation-house-sandbox", local.environment_management.account_ids["core-vpc-sandbox"]) : format("arn:aws:iam::%s:role/modernisation-account-limited-read-member-access", local.environment_management.modernisation_platform_account_id),
+      format("arn:aws:iam::%s:role/InstanceSchedulerLambdaFunctionPolicy", local.environment_management.account_ids["core-shared-services-production"])
```

Comment:
I don't think this is needed here, the OIDC provider you will be using is here - https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/core-shared-services/iam.tf#L203

Reply:
Ah I see you have it below, I think this change can be removed

Reply:
I wasn't sure, since it's going to iterate through other accounts... ?

Reply:
I shall remove it then. I can always raise another PR :D